project_task_description_template/models/task_template.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class TaskTemplate(models.Model):
+    _name = "project.task.template"
+
+    name = fields.Char(string="Name")
```

Comment:
Remove please `string` parameter. This parameter is duplicated field name

Reply:
okay, I'll fix it